index.js

```diff
@@ -0,0 +1,298 @@
+    let dest = path.join(options['build-dir'], 'export', source)
+    let dir = path.dirname(dest)
+    source = path.join(options['build-dir'], 'files', source)
+
+    logger(`Exporting ${source} to ${dest}`)
+    return mkdirs(dir)
+      .then(function () {
+        return copy(source, dest)
+      })
+  })
+  return Promise.all(copies)
+}
+
+function flatpakBuilder (options, finish) {
+  let args = []
+  addCommandLineOption(args, 'arch', options['arch'])
+  addCommandLineOption(args, 'force-clean', true)
+  addCommandLineOption(args, 'allow-missing-runtimes', true)
+  if (!finish) {
+    addCommandLineOption(args, 'build-only', true)
+  } else {
+    addCommandLineOption(args, 'finish-only', true)
+  }
+  args.concat(options['extra-flatpak-builder-args'])
+
+  args.push(options['build-dir'])
+  args.push(options['manifest-path'])
+  return spawnWithLogging(options, 'flatpak-builder', args)
+}
+
+function flatpakBuildExport (options, manifest) {
+  let args = ['build-export']
+  addCommandLineOption(args, 'arch', options['arch'])
+  addCommandLineOption(args, 'gpg-sign', options['gpg-sign'])
+  addCommandLineOption(args, 'gpg-homedir', options['gpg-homedir'])
+  addCommandLineOption(args, 'subject', options['subject'])
+  addCommandLineOption(args, 'body', options['body'])
+  if (options['build-runtime']) addCommandLineOption(args, 'runtime', true)
+  args.concat(options['extra-flatpak-build-export-args'])
+
+  args.push(options['repo-dir'])
+  args.push(options['build-dir'])
+  if (manifest['branch']) args.push(manifest['branch'])
+  return spawnWithLogging(options, 'flatpak', args)
+}
+
+function flatpakBuildBundle (options, manifest) {
+  if (!options['bundle-path']) return
+
+  let args = ['build-bundle']
+  addCommandLineOption(args, 'arch', options['arch'])
+  addCommandLineOption(args, 'gpg-keys', options['gpg-keys'])
+  addCommandLineOption(args, 'gpg-homedir', options['gpg-homedir'])
+  addCommandLineOption(args, 'repo-url', options['bundle-repo-url'])
+  if (options['build-runtime']) addCommandLineOption(args, 'runtime', true)
+  args.concat(options['extra-flatpak-build-bundle-args'])
+
+  args.push(options['repo-dir'])
+  args.push(options['bundle-path'])
+  args.push(manifest['id'])
+  if (manifest['branch']) args.push(manifest['branch'])
+  return mkdirs(path.dirname(options['bundle-path']))
+    .then(function () {
+      return spawnWithLogging(options, 'flatpak', args)
+    })
+}
+
+exports.bundle = function (manifest, options, callback) {
+  manifest = kebabify(manifest)
+  options = kebabify(options)
+  if (manifest['app-id']) manifest['id'] = manifest['app-id']
+
+  return ensureWorkingDir(options)
+    .then(() => {
+      options = getOptionsWithDefaults(options, manifest)
+      options.arch = flatpakifyArch(options.arch)
+
+      logger(`Using manifest...\n${JSON.stringify(manifest, null, '  ')}`)
+      logger(`Using options...\n${JSON.stringify(options, null, '  ')}`)
+    })
```

Comment:
Would be a nice touch to have a `ensureFlatpakVersion` that would let you know if you are using pre-0.6.13.